ark/utils/data_utils.py

```diff
@@ -205,6 +209,9 @@ def generate_pixel_cluster_mask(fovs, base_dir, seg_dir, pixel_consensus_dir,
             os.path.join(base_dir, pixel_consensus_dir, fov + '.feather')
         )
 
+        # ensure integer display and not float
+        fov_data[cluster_col] = fov_data[cluster_col].astype(int)
+
```

Comment:
Below, set option for segmentation file suffix (like you did before) instead of hardcoding '_feature_0.tif'. Also applies to genereate_cell_cluster_mask above.